### ISSUE TITLE
No data event

### ISSUE THREAD

**Issue by JasperFioole21** [Loghorn/ant-plus#20]
I do not get any data, I did open zwift which tells me my heartrate, so the dongle is doing its thing. I'm using OS X.

**Comment by JasperFioole21** [comment]
bump

**Comment by Loghorn** [comment]
Sorry, I was on vacation.
I don't have a mac, so I'm sorry but I cannot help you.
Did you try the samples? Did you get any error messages?

**Comment by JasperFioole21** [comment]
Hi, no problem. I did not get any error nor data.

**Comment by Loghorn** [comment]
Any output at all?

**Comment by JasperFioole21** [comment]
Non at all, so I have no idea where to look.

**Comment by Loghorn** [comment]
Wich sample are you trying?

**Comment by JasperFioole21** [comment]
I was trying this one:
https://github.com/Loghorn/ant-plus/blob/master/sample/sample.js

Do I need to send some kind of signal to the dongle to start receiving data?

**Comment by Loghorn** [comment]
What kind of dongle are you using?

**Comment by JasperFioole21** [comment]
I'll tell you tonight, my dongle is at home and I'm at work right now, thank you!

**Comment by JasperFioole21** [comment]
It's a garmin one.

**Comment by Loghorn** [comment]
Maybe you have some software running that exclusively locks the dongle?
I'm sorry, but I'm just guessing because I don't have a mac to test or debug...
I'll leave the issue open, just in case someone else can provide more help

**Comment by JasperFioole21** [comment]
Nope, I'd like to run it on a raspberry pi in future, so I will give it another shot when I have one.

**Comment by Loghorn** [comment]
On Linux, it works fine, so you should not have any issue with the raspberry :)
Btw, if you look at issue #2, you can find some people that successfully run it on mac
